analysed-packages/libpng/version-1.6.39/libpng-1.6.39-SPDX2TV.spdx

```diff
@@ -0,0 +1,10918 @@
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/contrib/gregbook/readpng.c
+SPDXID: SPDXRef-item494705
+FileChecksum: SHA1: ea1f8627785b1191de5a47226f419037fa8d34d8
+FileChecksum: SHA256: 890827fd946133628925099a1ba7ad79312c5a738553b3ccc6e6ca814298ca0b
+FileChecksum: MD5: e0dfe1a284fb2d0208770ba7db144549
+LicenseConcluded: LicenseRef-BSD-3-Clause-9049cb086704ce473d2cd47147fccf46 OR LicenseRef-GPL-2.0-or-later
+LicenseComments: <text> (The information in the file is:
+
+This software is provided "as is," without warranty of any kind,
+express or implied. In no event shall the author or contributors
+be held liable for any damages arising in any way from the use of
+this software.
+
+The contents of this file are DUAL-LICENSED. You may modify and/or
+redistribute this software according to the terms of one of the
+following two licenses (at your option):
+
+
+LICENSE 1 ("BSD-like with advertising clause"):
+
+Permission is granted to anyone to use this software for any purpose,
+including commercial applications, and to alter it and redistribute
+it freely, subject to the following restrictions:
+
+1. Redistributions of source code must retain the above copyright
+notice, disclaimer, and this list of conditions.
+2. Redistributions in binary form must reproduce the above copyright
+notice, disclaimer, and this list of conditions in the documenta-
+tion and/or other materials provided with the distribution.
+3. All advertising materials mentioning features or use of this
+software must display the following acknowledgment:
+
+This product includes software developed by Greg Roelofs
+and contributors for the book, "PNG: The Definitive Guide,"
+published by O'Reilly and Associates.
+
+
+LICENSE 2 (GNU GPL v2 or later):
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+This is a dual license case. We concluded BSD-3-Clause OR GPL-2.0+) AND NOASSERTION AND NOASSERTION </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-Dual-license
+LicenseInfoInFile: LicenseRef-BSD-style
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-greg-roelofs
+FileCopyrightText: <text> Copyright (c) 1998-2007,2017 Greg Roelofs. All rights reserved. </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/contrib/gregbook/LICENSE
```

Comment:
Here you have concluded LicenseRef-BSD-3-Clause-9049cb086704ce473d2cd47147fccf46 OR LicenseRef-GPL-2.0-or-later but this is not the license of this file